servo/connectors/kubernetes.py

```diff
@@ -2813,7 +2839,10 @@ async def destroy(self, error: Optional[Exception] = None) -> None:
 
         self.logger.info(f'destroyed tuning Pod "{self.name}"')
 
-    async def handle_error(self, error: Exception, mode: "FailureMode") -> bool:
+    async def handle_error(self, error: Exception, mode: "FailureMode" = None) -> bool:
+        if mode is None:
+            mode = self.on_failure
+
```

Comment:
Why didn't you just yank the `mode` param entirely?

Reply:
I thought it would allow greater flexibility, I'll push an update to remove it